### DISCUSSION TITLE
Black bars in window mode (but not in full-screen mode)

### DISCUSSION THREAD

**Discussion by UMP45** [smplayer-dev/smplayer#824] [General]
I get black bars on both sides on some videos (not all videos) in window mode: 

![Clipboard01](https://github.com/smplayer-dev/smplayer/assets/12972788/8c715551-6fc5-4a0a-8424-325c8eaa6ddc)

In full-screen mode, the video is fine:

![Clipboard02](https://github.com/smplayer-dev/smplayer/assets/12972788/0bd51bb2-dda5-45ab-89bb-408cf88869be)

Video info: 
> Initial Video Stream
> Resolution: 1920 x 1080
> Aspect ratio: 1.77778
> Format: hevc

My OS: Windows 10 Home (22H2) (build 19045.3570)
![image](https://github.com/smplayer-dev/smplayer/assets/12972788/28637574-819b-4651-bf9c-e0be4361d8ca)

**Comment by smplayer-dev** [comment]
I think it's normal; probably the top and bottom bars are part of the image, and SMPlayer adds side bars to maintain the aspect ratio. You can use the 'Zoom' option in the Video menu.
